examples/messaging/jms-websocket-mp/src/main/java/io/helidon/messaging/mp/example/WebSocketEndpoint.java

```diff
@@ -0,0 +1,96 @@
+
+/*
+ * Copyright (c)  2020 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.helidon.messaging.mp.example;
```

Comment:
The package structure should be:
`io.helidion.examples.messaging.mp` 
The existing examples have it wrong, this would follow our naming convention.

Reply:
fixed